src/pages/library/DataBrowserPreview.js

```diff
@@ -67,6 +82,21 @@ const DataBrowserPreview = ({ id }) => {
     loadTable()
   }
 
+  const formatTableCell = cellContent => {
+    return Utils.cond(
+      [!Utils.cantBeNumber(cellContent), () => cellContent],
+      [Utils.maybeParseJSON(cellContent), () => {
```

Comment:
I wonder about this approach.  this may be a little over-eager.  For instance, `"test"` would pass as a json string (since it's quoted).  I do also wonder about a potential perf impact of doing this in the cell since you could have a table full of big json fields.  Might make sense to do it lazily

Reply:
Running `JSON.parse("test")` will fail - I'm trying to catch cases where they would successfully render as a smaller field above it ('cant be number'), because `JSON.parse(123)` will succeed... I can add extra cases as necessary!

I was a little concerned about the perf after it was brought up today in stand up, but then, I'm not sure how to lazily test for json - We would have to run JSON.parse for all fields to test them to determine if a json button should be shown, right? We never run the jsonparsing again after that. If there's a better way to test if a variable is a json object, we could try that, but from what I found on stackoverflow, it seems like parsing is the best way

Reply:
Oh, I was thinking about `JSON.parse('"test"')`

That being said, it's not a huge deal.  I think to do it lazily would require a different ux.  Maybe it's something like a context menu or something that happens on hover.